one-ui/src/test/java/com/marklogic/hub/oneui/TestHelper.java

```diff
@@ -73,17 +74,26 @@ public TestHelper() throws IOException {
     }
 
     public void authenticateSession() {
-        createUser(dataHubDeveloperUsername,dataHubDeveloperPassword,"data-hub-developer");
         EnvironmentInfo environmentInfo = new EnvironmentInfo(mlHost, "DIGEST", 8000,"DIGEST", 8002,"DIGEST", 8010, "DIGEST", 8011);
         hubConfig.setCredentials(environmentInfo, dataHubDeveloperUsername, dataHubDeveloperPassword);
     }
 
     public void authenticateSessionAsEnvironmentManager() {
-        createUser(dataHubEnvironmentManagerUsername,dataHubEnvironmentManagerPassword,"data-hub-environment-manager");
         EnvironmentInfo environmentInfo = new EnvironmentInfo(mlHost, "DIGEST", 8000,"DIGEST", 8002,"DIGEST", 8010, "DIGEST", 8011);
         hubConfig.setCredentials(environmentInfo, dataHubEnvironmentManagerUsername, dataHubEnvironmentManagerPassword);
     }
 
+    public void authenticateSessionAsAdmin() {
+        EnvironmentInfo environmentInfo = new EnvironmentInfo(mlHost, "DIGEST", 8000,"DIGEST", 8002,"DIGEST", 8010, "DIGEST", 8011);
+        hubConfig.setCredentials(environmentInfo, adminUserName, adminPassword);
+    }
+
+    @PostConstruct
+    public void createTestUsers() {
+        createUser(dataHubDeveloperUsername,dataHubDeveloperPassword,"data-hub-developer");
+        createUser(dataHubEnvironmentManagerUsername,dataHubEnvironmentManagerPassword,"data-hub-environment-manager");
```

Comment:
I think that if we need another user, it should be created in the Installer program as well.